### DISCUSSION TITLE
SystemD Service Sample

### DISCUSSION THREAD

**Discussion by alidehghan** [goreleaser/goreleaser#5031] [Q&A] [answered]
How can I build a deb package with systemD service ability? What I need is building (releasing) a deb package from go binary (or source, no matter) in which when user install, a systemd service is automatically installed and runs (like many usual app packages, for instance nginx)

**Comment by caarlos0** [comment, marked as answer]
you'll need to have the systemd unit added to your nfpm configuration, and, a post-install script to activate it

some docs that might help: https://nfpm.goreleaser.com/tips/#systemd-and-upstartinit

**Reply by meskio** [reply]
The link doesn't work, I guess it has been moved to: https://nfpm.goreleaser.com/docs/tips/#systemd-and-upstartinit
